optimum/onnxruntime/trainer.py

```diff
@@ -764,7 +762,7 @@ def evaluation_loop_ort(
         Works both with or without labels.
         """
         logger.info("[INFO] ONNX Runtime inference starts...")
-        self.infer_sess = None
+        self.ort_model = None
```

Comment:
Is it necessary for `ort_model` to be an attribute?

Reply:
Nope, it would be enough to take it as a function variable. Good point, will correct this.